packages/portal/src/app/shared/card/card.component.html

```diff
@@ -1,5 +1,5 @@
 <section>
-  <mat-card class="loading-background">
+  <mat-card class="loading-background background-hover">
     <img
```

Comment:
To preserve semantics I would avoid adding a class with a hover suffix, unless it's added dynamically while hovering. 
I would recommend you add a second class called just `card` for example, and add the animation to the hover and to the focus pseudos. The reason why we do this is so when browsing with the keyboard or other assistive technology, it gets also enabled.

Reply:
@anfibiacreativa thank you for guiding me , i will work on it ,